tools/driver-cxx/driver-cxx.cpp

```diff
@@ -206,7 +215,23 @@ int main(int argc, char **argv) {
     filePathFilter->include(regex);
   }
 
-  if (junkDetectionEnabled) {
+  if (tool::EnableAST && compilationDatabaseInfoAvailable) {
+    std::vector<mull::MutatorKind> mutationKinds;
+    for (auto &mutator : mutatorsOptions.mutators()) {
+      mutationKinds.push_back(mutator->mutatorKind());
+    }
+
+    mull::MutatorKindSet mutatorKindSet = mull::MutatorKindSet::create(mutationKinds);
+
+    mull::ASTFinder astFinder(mutatorKindSet);
+    astFinder.findMutations(diagnostics, configuration, program, *filePathFilter, astStorage);
```

Comment:
Eventually, you may want to move it to a later stage and run only against functions covered with tests.

Reply:
I fully agree. Two additional comments:

1) I expected that you would suggest me to move this to the Driver code anyway and I am happy to do that later.
2) Given that we will use white search combined with whitelisting of the paths the difference between what is being run and where the mutations are found should be as dramatic as before.